### DISCUSSION TITLE
Gravity Forms 2.5 support

### DISCUSSION THREAD

**Discussion by lewebsimple** [harness-software/wp-graphql-gravity-forms#106] [General]
At first glance, it seems this plugin works with the latest Gravity Forms 2.5 version, I'm opening this discussion to see what new features could be implemented (new layouts?).

**Comment by justlevine** [comment]
@lewebsimple what new layouts are you referring to?
GF's documentation is abysmal, but I'm pretty sure we caught everything added in 2.5....
